packages/css/src/components/grid/_mixins.scss

```diff
@@ -0,0 +1,110 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright Gemeente Amsterdam
+ */
+
+@mixin ams-grid {
```

Comment:
> For what reason?

Ease of use mostly. Currently, I can't see at a glance what `.ams-grid` does, I have to go to another file and connect the CSS block with the right class in my head. I quite often browse through the Utrecht CSS and this gets a little annoying.

On the other hand, having both in the same files makes that file longer, so that might decrease the readability.

> It also allows consumers to import the mixins without the class names.

This also works when both are in the same file, no? You can import mixins without importing the built CSS?